service/controller/resource/cproutetables/resource.go

```diff
@@ -44,6 +44,11 @@ func New(config Config) (*Resource, error) {
 		return nil, microerror.Maskf(invalidConfigError, "%T.Installation must not be empty", config)
 	}
 
+	// TODO: This is temporary fix until we get rid of routeTables flag.
```

Comment:
Note: remember to write something more informative here or fix it in config parsing where it's split by `strings.Split`